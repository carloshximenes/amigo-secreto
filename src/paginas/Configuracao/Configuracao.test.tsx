import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";
import Configuracao from ".";

jest.mock("../../state/hooks/useListarParticipantes", () => {
	return {
		useListarParticipantes: jest.fn(),
	};
});

const mockDeNavegacao = jest.fn();

jest.mock("react-router-dom", () => {
	return {
		useNavigate: () => mockDeNavegacao,
	};
});

describe("a pagina de configuracao", () => {
	test("deve ser renderizada corretamente", () => {
		const { container } = render(
			<RecoilRoot>
				<Configuracao />
			</RecoilRoot>
		);

		expect(container).toMatchSnapshot();
	});
});
