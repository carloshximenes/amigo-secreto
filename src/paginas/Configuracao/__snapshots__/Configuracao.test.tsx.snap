// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a pagina de configuracao deve ser renderizada corretamente 1`] = `
<div>
  <div
    class="card"
  >
    <section>
      <h2>
        Vamos Come√ßar!
      </h2>
      <form>
        <div
          class="grupo-input-btn"
        >
          <input
            placeholder="Insira o nome dos participantes"
          />
          <button
            disabled=""
          >
            Adicionar
          </button>
        </div>
        
      </form>
      <ul />
      <footer
        class="rodape-configuracoes"
      >
        <button
          class="botao"
        >
          Iniciar Brincadeira
        </button>
        <img
          alt="Sacolas de compras"
          src="/imagens/sacolas.png"
        />
      </footer>
    </section>
  </div>
</div>
`;
